Library/Homebrew/cmd/uses.rb

```diff
@@ -32,12 +32,7 @@ def uses_args
              description: "Include all formulae that specify <formula> as `:optional` type dependency."
       switch "--skip-recommended",
              description: "Skip all formulae that specify <formula> as `:recommended` type dependency."
-      switch "--devel",
-             description: "Show usage of <formula> by development builds."
-      switch "--HEAD",
-             description: "Show usage of <formula> by HEAD builds."
```

Comment:
Might want to check for these and output an `odeprecated` instead?

Reply:
Should we keep `conflicts "--devel", "--HEAD"`?